pages/index.tsx

```diff
@@ -1,23 +1,65 @@
 import type { NextPage } from 'next';
 import Image from 'next/image';
 import Head from 'next/head';
-import styles from '../styles/Home.module.css';
+import Link from 'next/link';
+import styles from '../styles/Home.module.scss';
+import TopCTA from '../components/TopCTA';
+import BottomCTA from '../components/BottomCTA';
 
 const Home: NextPage = () => {
     return (
         <div className={styles.container}>
             <Head>
                 <title>Are we headless yet? | Wagtail</title>
```

Comment:
**Suggestion**: Maybe this is needed in a separate ticket, but we should add some social media metadata tags here ([CSS Tricks](https://css-tricks.com/essential-meta-tags-social-media/)) and maybe use a [favicon generator](https://realfavicongenerator.net/) to make all the favicon files?

Reply:
If you're familiar with Codebase you could copy paste the above into a new ticket 🙂